### DISCUSSION TITLE
Where we are, where we going?

### DISCUSSION THREAD

**Discussion by navidcy** [dhruvbhagtani/sfc-perturbation-expts#4] [General]
@dhruvbhagtani has already made a start by putting down some potential experiments in the repo's [`README.md`](https://github.com/dhruvbhagtani/varying-surface-forcing/blob/main/README.md).

As far as I view the current status quo we have sort of two main questions in mind:

Projects:
- Gyre response
- Overturning circulation response
- ...?

The two projects above are not completely independent of one another of course. 

Further more we have the following "tools" to address these:

Models:
- 1 deg RYF
- 0.25 deg RYF
- 0.10 deg RYF
- ...?

using, of course, different forcing protocols, tweaking the boundary-layer scheme, and what not.

So, perhaps a table (or something else) with potential/proposed experiments may include:
- short _descriptive_ name
- project it falls under (perhaps both!)
- model used
- forcing protocol
- main hypothesis/question that inspired this experiment (e.g., "whether Southern Ocean winds controls the North Atlantic sub polar gyre strength")
- first things we should look (e.g., "look how the strength of the sub polar gyre changed in this experiment compared to `control` or other experiments")
- ...?

On second read, I think that a table might become cumbersome. Perhaps just a small paragraph for each experiment with 1 sentences under 4-5 bullet points?

Somehow we should summarise the results we have up to now. Perhaps the first four entries of this table/list should describe the results we have so far.

 What do you all think?

cc @dhruvbhagtani, @rmholmes,  @AndyHoggANU

**Comment by navidcy** [comment]
> On second read, I think that a table might become cumbersome. Perhaps just a small paragraph for each experiment with 1 sentences under 4-5 bullet points?

This is what I mean by "cumbersome"...

<img width="250" alt="Screen Shot 2021-10-05 at 7 17 41 am" src="https://user-images.githubusercontent.com/7112768/135919067-3bad3e35-ec59-49fc-9d98-71dae88afd49.png">

**Reply by dhruvbhagtani** [reply]
Yes @navidcy, the present readme looks quite cumbersome. Your strategy sounds much more appealing. In an ideal world, we could also draw a flow chart, which may be perfect for explaining the motivation behind our experiments. Something like:

We began with Ryan's no-stress runs, realised they had very shallow KPP depths, then we made a parameterisation for resolved velocity shear, then we had less overturning which resulted in lesser meridional heat flux, and so on...

Some comments:
1. I think in each of the listed expts, we'll be looking at both overturning and gyre circulation, so should we really need two projects?
2. By forcing, you mean RYF/IAF? I think that's a nice addition, as we could maybe mention whether we have the forcing everywhere, or nowhere, or in select spatial locations.
3. We could also club motivation and "first things we should look", as they will more or less be intertwined.

I'll reframe the README file and add the list of experiments we have completed.

**Reply by dhruvbhagtani** [reply]
I added some project cards in the "gyre response" section, but I don't want to drift off if I am doing it wrong. Can someone check, and give suggestions before I write some things for the other runs?

**Reply by rmholmes** [reply]
Having now looked around a bit, I have the following initial comments.

I agree with @dhruvbhagtani that there is no need to separate our analyses/focus between gyre and overturning. We will look at both. In fact, they are closely linked. E.g. The subtropical gyres are a significant contributor to the density-latitude overturning streamfunction.

A list of proposed (and completed!) experiments is a good idea, with Navid's dot points. I would focus on "main hypothesis/question" and "forcing protocol" as these are the most important factors to get right (why are we doing it, and can we/how can we do it). By forcing protocol here we mean how we will perturb the RYF forcing. I don't see the benefit of IAF at this point.

The problem with project cards appears to be that we can't comment on them individually (at least I couldn't figure out how)? What we really want is an easily collapsable list of experiments that we can comment on individually. A cumbersome option would be an issue per experiment. Is there another approach?

**Reply by navidcy** [reply]
OK, agreed regarding no need to separate Gyre and Overturning response in different projects -- I merged the two.
